### PR TITLE
Add a new section "secret management" to gds-way, containing 4 articles

### PR DESCRIPTION
**Pull request by venusbb**
## Add a new section "secret management" to gds-way, containing the first 4 articles:

- secret auditing
- access control list
- pre-commit hook
- principle of least access

Venus B and Ollie C. has been writing the articles together with input from the tech community and have worked with tech writer to modify the content as well as two eyed. 

## To review
Run the PR locally, check the navigation bar, content and hyperlinks.

### CODE REVIEW

File: source/standards/pre-commit-hooks.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: Git Pre-commit Hooks
+last_reviewed_on: 2020-11-27
+review_in: 6 months
+---
+# <%= current_page.data.title %>
+
+[Git hooks](https://githooks.com/) are scripts that Git executes before or after events such as commit, push, and receive. Pre-commit hooks are useful in secrets management because they check for errors and problems before developers commit work locally on their machines.
+
+The Cyber Security team are working on a [pre-commit hook](https://github.com/alphagov/gds-pre-commit) for GDS to use. The main reason for using pre-commit hooks is to prevent secrets being pushed to GitHub repositories.
```
Comment (0atman) [inline source/standards/pre-commit-hooks.html.md.erb:10]: It is more correct to say that we have a reccomended tool. See the language of https://github.com/alphagov/gds-pre-commit/blob/master/README.md

Our advice has changed recently, we USED to build a pre-commit hook that offered turnkey support, however the maintenance and support burden was too high, we've now tweaked our repo to provide guidance on how to use off-the-shelf tools.

**Comment by 0atman** [comment]
Thanks Venus! I've added a comment about the pre-commit section. Let me know if I can be of any more help!

File: source/standards/pre-commit-hooks.html.md.erb (hunk 1 above)
Comment (venusbb) [reply source/standards/pre-commit-hooks.html.md.erb]: I have changed it according to your recommendations! Could you have another look please? 

Happy new year!

**Review by 0atman — DISMISSED** [review]
wording perfect

File: source/standards/pre-commit-hooks.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+---
+# <%= current_page.data.title %>
+
+[Git hooks](https://githooks.com/) are scripts that Git executes before or after events such as commit, push, and receive. Pre-commit hooks are useful in detecting secrets automatically before developers commit work locally on their machines.
+
+The Cyber Security team has recommended a pre-commit git hook called [Detect Secrets](https://github.com/Yelp/detect-secrets) for preventing secrets being commited to git.
+
+## Further guidance
+- [Git hooks](https://githooks.com/)
+- [README on Cyber Security gds-pre-commit repository](https://github.com/alphagov/gds-pre-commit)
```
Comment (0atman) [inline source/standards/pre-commit-hooks.html.md.erb:14]: don't want to block the PR, but this should say:

```suggestion
- [Cyber Security pre-commit best practices](https://github.com/alphagov/gds-pre-commit)
```

File: source/standards/pre-commit-hooks.html.md.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+---
+# <%= current_page.data.title %>
+
+[Git hooks](https://githooks.com/) are scripts that Git executes before or after events such as commit, push, and receive. Pre-commit hooks are useful in detecting secrets automatically before developers commit work locally on their machines.
+
+The Cyber Security team has recommended a pre-commit git hook called [Detect Secrets](https://github.com/Yelp/detect-secrets) for preventing secrets being commited to git.
+
+## Further guidance
+- [Git hooks](https://githooks.com/)
+- [Cyber Security pre-commit install guidance](https://github.com/alphagov/gds-pre-commit)
```
Comment (0atman) [inline source/standards/pre-commit-hooks.html.md.erb:14]: fantastic

**Review by 0atman — APPROVED** [review]

**Pull request merged**